fabtests/functional/inj_complete.c

```diff
@@ -188,11 +183,9 @@ int main(int argc, char **argv)
 			break;
 		case '?':
 		case 'h':
-			ft_usage(argv[0], "FI_Inject_Completion Functional Test");
-			FT_PRINT_OPTS_USAGE("-m <size>", "size of Injection message");
-			FT_PRINT_OPTS_USAGE("-S", "enable testing with fi_sendmsg");
+			ft_csusage(argv[0], "FI_Inject_Completion Functional Test");
+			FT_PRINT_OPTS_USAGE("-N", "enable testing with fi_sendmsg");
 			FT_PRINT_OPTS_USAGE("-R", "enable testing with fi_recvmsg");
```

Comment:
Out of curiosity, why is there an fi_recvmsg test option?  That seems unrelated to the purpose of the test.

Reply:
fi_recvmsg was not needed and I removed that and related code in a separate commit.